### PR TITLE
PWN-3266 - Show user card in gallery

### PR DESCRIPTION
**Pull request by kuznetsov**
### Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3266

### Description of Work

- Save user card image to private `Pictures` directory
- Add saved image to gallery

### Screenshots

Screenshot 2022-05-04 at 17 01 55

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt [hunk 1]
```diff
@@ -44,7 +45,7 @@ class FileRepository(private val context: Context) {
         val fileName = name ?: Date().toString()
         try {
             val mainDir = File(
-                context.getExternalFilesDir(Environment.DIRECTORY_PICTURES),
+                Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES),
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt:48]: let's do Dependency Injection here, shall we? would be much easier later on

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt:48]: Are you sure it's working on all OS since this method is deprecated?

Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt:48]: It works on latest OS

Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt:48]: We will test on earlier version on QA

**Review by gslevinkov — APPROVED** [review]

**Review by davran1001 — APPROVED** [review]

**Pull request merged**